Kconfig

```diff
@@ -2007,6 +2007,12 @@ menu "LVGL configuration"
 			select LV_FONT_MONTSERRAT_24
 			select LV_FONT_MONTSERRAT_26
 			select LV_USE_DEMO_WIDGETS
+
+		config LV_DEMO_BENCHMARK_FREETYPE_FONT_PATH
+			string "Path to font file which is used in benchmark test"
+			default "./lvgl/tests/src/test_files/fonts/noto/NotoSansSC-Regular.ttf"
+			depends on LV_USE_DEMO_BENCHMARK && (LV_USE_FREETYPE || LV_USE_TINY_TTF)
```

Comment:
tinyttf cannot use the `./` path of freetype. You can add an independent path configuration `LV_DEMO_BENCHMARK_TINY_TTF_FONT_PATH` to tinyttf.